pkg/core/localize/locales/en/cmd/rule.en.toml

```diff
@@ -3,47 +3,48 @@ one = 'Manage artifact rules in a Service Registry instance'
+
+For more information about supported Service Registry content and rules, see https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1/guide/9b0fdf14-f0d6-4d7f-8637-3ac9e2069817.
 '''
 
 [registry.rule.cmd.example]
 one = '''
-## Enable global compatibility rule for artifacts of the current Service Registry instance
+## Enable the global compatibility rule for all artifacts in the current Service Registry instance
 $ rhoas service-registry rule enable --rule-type=compatibility --config=full
 
-## Enable global compatibility rule for artifacts of a specific Service Registry instance
-$ rhoas service-registry rule enable --rule-type=compatibility --config=full-transitive --instance-id 8ecff228-1ffe-4cf5-b38b-55223885ee00
+## Enable the global compatibility rule for all artifacts in a specific Service Registry instance
+$ rhoas service-registry rule enable --rule-type=compatibility --config=full-transitive --instance-id=8ecff228-1ffe-4cf5-b38b-55223885ee00
 
-## display configuration details of global validity rule for current Service Registry instance
+## Display the configuration details of the global validity rule for the current Service Registry instance
 $ rhoas service-registry rule describe --rule-type=validity
 '''
 
 [registry.rule.enable.cmd.description.short]
 one='Enable validity and compatibility rules'
 
 [registry.rule.enable.cmd.description.long]
-one='Enable validity and compatibility rules for the specified Service Registry instance or artifact'
+one='Enable validity and compatibility rules for the specified Service Registry instance or artifact.'
 
 [registry.rule.enable.cmd.example]
 one='''
-## Enable global compatibility rule for artifacts of the current Service Registry instance
+## Enable the global compatibility rule for all artifacts in the current Service Registry instance
 $ rhoas service-registry rule enable --rule-type=compatibility --config=full
 
-## Enable global compatibility rule for artifacts of a specific Service Registry instance
-$ rhoas service-registry rule enable --rule-type=compatibility --config=full-transitive --instance-id 8ecff228-1ffe-4cf5-b38b-55223885ee00
+## Enable the global compatibility rule for all artifacts in a specific Service Registry instance
+$ rhoas service-registry rule enable --rule-type=compatibility --config=full-transitive --instance-id=8ecff228-1ffe-4cf5-b38b-55223885ee00
 
-## Enable validity rule for a specific artifact
+## Enable the validity rule for a specific artifact
 $ rhoas service-registry rule enable --rule-type=validity --config=syntax-only --artifact-id=my-artifact
 '''
 
 [registry.rule.enable.log.info.enabling.globalRules]
-one='Enabling global "{{.RuleType}}" rule for artifacts of Service Registry instance with configuration "{{.Configuration}}"'
+one='Enabling global "{{.RuleType}}" rule with configuration "{{.Configuration}}" for artifacts in the Service Registry instance with ID "{{.ID}}"'
```

Comment:
This will need changes in the PR, I will be happy to push changes to this PR.
```
> ./rhoas service-registry rule enable --rule-type validity --config full
Enabling global "validity" rule with configuration "full" for artifacts in the Service Registry instance with ID "<no value>"
```

Reply:
My apologies -- I saw those variables used elsewhere in the file so I thought they could be used anywhere, I didn't mean to create extra work for you!